examples/static_meshlets/shaders/meshlet.task

```diff
@@ -0,0 +1,89 @@
+#version 460
+#extension GL_EXT_shader_16bit_storage: require
+#extension GL_EXT_shader_8bit_storage: require
+#extension GL_NV_mesh_shader : require
+#extension GL_EXT_nonuniform_qualifier : require
+#extension GL_GOOGLE_include_directive : enable
+#extension GL_ARB_shader_draw_parameters: require
+#extension GL_KHR_shader_subgroup_ballot: require
```

Comment:
```suggestion
#extension GL_EXT_shader_16bit_storage : require
#extension GL_EXT_shader_8bit_storage : require
#extension GL_NV_mesh_shader : require
#extension GL_EXT_nonuniform_qualifier : require
#extension GL_GOOGLE_include_directive : enable
#extension GL_ARB_shader_draw_parameters : require
#extension GL_KHR_shader_subgroup_ballot : require
```

^ Just to be consistent. I'd be fine with `a: b` as well, it shall just be the same everywhere.